quarkus/admin/distribution/LICENSE

```diff
@@ -207,1314 +207,898 @@ This binary artifact bundles the following projects:
-
---------------------------------------------------------------------------------
-
 Group: org.apache.iceberg Name: iceberg-core Version: 1.9.0
-Project URL (from POM): https://iceberg.apache.org/
-License (from POM): Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
-
---------------------------------------------------------------------------------
-
 Group: org.apache.iceberg Name: iceberg-gcp Version: 1.9.0
-Project URL (from POM): https://iceberg.apache.org/
-License (from POM): Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
+Project URL: https://iceberg.apache.org/
+License: Apache License 2.0 - https://www.apache.org/licenses/LICENSE-2.0.txt
 
 --------------------------------------------------------------------------------
 
 Group: org.checkerframework Name: checker-qual Version: 3.49.1
-Project URL (from POM): https://checkerframework.org/
-License (from POM): MIT License - http://opensource.org/licenses/MIT
+Project URL: https://checkerframework.org/
+License: MIT License
+| The Checker Framework
+| Copyright 2004-present by the Checker Framework developers
+|
+|
+| Most of the Checker Framework is licensed under the GNU General Public
+| License, version 2 (GPL2), with the classpath exception.  The text of this
+| license appears below.  This is the same license used for OpenJDK.
+|
+| A few parts of the Checker Framework have more permissive licenses, notably
+| the parts that you might want to include with your own program.
+|
+|  * The annotations and utility files are licensed under the MIT License.
+|    (The text of this license also appears below.)  This applies to
+|    checker-qual*.jar and checker-util.jar and all the files that appear in
+|    them, which is all files in checker-qual and checker-util directories.
+|    It also applies to the cleanroom implementations of
+|    third-party annotations (in checker/src/testannotations/,
+|    framework/src/main/java/org/jmlspecs/, and
+|    framework/src/main/java/com/google/).
+|
+| The Checker Framework includes annotations for some libraries.  Those in
+| .astub files use the MIT License.  Those in https://github.com/typetools/jdk
+| (which appears in the annotated-jdk directory of file checker.jar) use the
+| GPL2 license.
+|
+| Some external libraries that are included with the Checker Framework
+| distribution have different licenses.  Here are some examples.
+|
+|  * JavaParser is dual licensed under the LGPL or the Apache license -- you
+|    may use it under whichever one you want.  (The JavaParser source code
+|    contains a file with the text of the GPL, but it is not clear why, since
+|    JavaParser does not use the GPL.)  See
+|    https://github.com/typetools/stubparser .
+|
+|  * Annotation Tools (https://github.com/typetools/annotation-tools) uses
+|    the MIT license.
+|
+|  * Libraries in plume-lib (https://github.com/plume-lib/) are licensed
+|    under the MIT License.
```

Comment:
You might want to check this.

Reply:
You mean what's actually bundled right ? If plume libs are not bundle, no need to mention it, right ?

Reply:
I was more concerned about "Most of the Checker Framework is licensed under the GNU General Public license".

Reply:
As you can see in the notice, checker-qual (the bundled artifact) is under MIT license.

Reply:
I included the complete LICENSE, it mentions MIT for checker-qual (not sure if we can just copy part of the LICENSE)

Reply:
I updated to use https://github.com/typetools/checker-framework/blob/master/LICENSE.txt#L393